plugin.audio.hearthis_at/resources/language/English/strings.po

```diff
@@ -0,0 +1,116 @@
+#
```

Comment:
can you please adapt the folder layout as well please?

this should be `resources/language/resource.language.en_gb/strings.po`

the german one should be  `resources/language/resource.language.de_de/strings.po`